packages/core/admin/shared/utils/session-auth.ts

```diff
@@ -0,0 +1,97 @@
+import crypto from 'crypto';
+import type { Modules } from '@strapi/types';
+
+export const REFRESH_COOKIE_NAME = 'strapi_admin_refresh';
+
+const DEFAULT_ACCESS_TOKEN_LIFESPAN = 10 * 60;
```

Comment:
This may have revealed an existing issue; DEFAULT_ACCESS_TOKEN_LIFESPAN is not used.

But we may also still want to export these, in case someone needs access to them from outside. It might be useful to know the defaults.